### PR TITLE
Improve Arabic Translation

### PR DESCRIPTION
**Pull request by mabehiry**
Added some translations

### CODE REVIEW

**Comment by mabehiry** [comment]
Why did it fail?
Is there any error in my file?

**Comment by w20k** [comment]
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

File: modules/backend/lang/ar/lang.php [hunk 1]
```diff
@@ -51,11 +78,73 @@
...
+        'reload' => 'إعادة تحميل',
+        'complete' => 'تم',
+        'select' => 'تحديد',
+        'select_all' => 'تحديد الكل',
+        'select_none' => 'إلغاء التحديد',
+        'select_placeholder' => 'من فضلك اختار',
+        'insert_row' => 'إضافة سجل',
+        'return_to_list' => 'عودة للقائمة'
+    ],
+		'relation' => [
```
Comment (w20k) [inline modules/backend/lang/ar/lang.php]: @mabehiry fix spaces! Should be alined!

Comment (mabehiry) [reply modules/backend/lang/ar/lang.php]: Done. Sorry for that

File: modules/backend/lang/ar/lang.php [hunk 2]
```diff
@@ -51,11 +78,73 @@
...
+        'reload' => 'إعادة تحميل',
+        'complete' => 'تم',
+        'select' => 'تحديد',
+        'select_all' => 'تحديد الكل',
+        'select_none' => 'إلغاء التحديد',
+        'select_placeholder' => 'من فضلك اختار',
+        'insert_row' => 'إضافة سجل',
+        'return_to_list' => 'عودة للقائمة'
+    ],
+	'relation' => [
```
Comment (LukeTowers) [inline modules/backend/lang/ar/lang.php]: Indentation is wrong here

Comment (mabehiry) [reply modules/backend/lang/ar/lang.php]: Done

**Pull request merged**